lntest/harness.go

```diff
@@ -358,6 +363,7 @@ func (n *NetworkHarness) newNode(name string, extraArgs []string,
 		BackendCfg: n.BackendCfg,
 		NetParams:  n.netParams,
 		ExtraArgs:  extraArgs,
+		Etcd:       n.useEtcd,
```

Comment:
So accept it as an arg here, and in the public `NewNode` instead.

Reply:
See above comment.